### ISSUE TITLE
retreiveSAM is spelled wrong

### ISSUE THREAD

**Issue by jforbess** [Sandia-Labs/PVLIB_Python#41]
Just a suggestion that it would be great if the function retreiveSAM were updated to use the correct spelling of retrieve, at least in the develop branch for future use.

Sorry if this is pendantic, but it's amazingly distracting to me.

**Comment by wholmgren** [comment]
Agreed. We fixed this in our fork awhile ago, but I don't that think we documented the change. Making a note of it here for when we integrate the fork back into the main project.

https://github.com/UARENForecasting/PVLIB_Python/blob/master/pvlib/pvsystem.py#L462

Now that I look at it again there's a typo in the doc string too.

If we wanted to be careful and sensitive to anybody that's already using it we could alias the old function and throw a warning, but I don't think that there's a large enough user base to worry about it at this point.
